langchain4j-vearch/src/test/java/dev/langchain4j/store/embedding/vearch/TestVearchClient.java

```diff
@@ -0,0 +1,61 @@
+package dev.langchain4j.store.embedding.vearch;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import lombok.Builder;
+import okhttp3.OkHttpClient;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import static com.google.gson.FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES;
+
+public class TestVearchClient {
+
+    private static final Gson GSON = new GsonBuilder()
+            .setFieldNamingPolicy(LOWER_CASE_WITH_UNDERSCORES)
+            .create();
+
+    private final TestVearchApi vearchApi;
+
+    @Builder
+    public TestVearchClient(String baseUrl, Duration timeout) {
+        OkHttpClient okHttpClient = new OkHttpClient.Builder()
+                .callTimeout(timeout)
+                .connectTimeout(timeout)
+                .readTimeout(timeout)
+                .writeTimeout(timeout)
+                .build();
+
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl(baseUrl)
+                .client(okHttpClient)
+                .addConverterFactory(GsonConverterFactory.create(GSON))
+                .build();
+
+        vearchApi = retrofit.create(TestVearchApi.class);
+    }
+
+    public void deleteSpace(String databaseName, String spaceName) {
```

Comment:
Can users delete space in vearch somehow? Is there a UI or something?
If not, maybe we should move this from tests to prod code?

Reply:
yeah, might be handy?

Reply:
got it

Reply:
done